js/main.js

```diff
@@ -15,19 +16,35 @@ $form.addEventListener('submit', handleSubmit);
 
 function handleSubmit(event) {
   event.preventDefault();
-  var newEntry = {
-    entryTitle: $title.value,
-    entryPhotoURL: $photoURL.value,
-    entryNotes: $notes.value,
-    entry: data.nextEntryId
-  };
-  data.nextEntryId++;
-  data.entries.unshift(newEntry);
-  $placeholderImg.setAttribute('src', 'images/placeholder-image-square.jpg');
-  $form.reset();
-  var $ul = document.querySelector('ul');
-  var newEntryDomTree = createNewEntryLi(newEntry);
-  $ul.prepend(newEntryDomTree);
+  if ($h2NewEntriesTitle.className === '') {
```

Comment:
Instead of basing this behavior on the class name of a header, you should base it on if the value of the `data.editing` is `null` or not.